wear/build.gradle.kts

```diff
@@ -67,6 +67,7 @@ android {
 
 dependencies {
     implementation(project(":common"))
+    implementation(libs.mpgraph)
```

Comment:
why do we need to include the module in Wear OS if its not used there?

Reply:
Otherwise it wont compile